### ISSUE TITLE
Marker not showing in the map

### ISSUE THREAD

**Issue by saepul8** [tananaev/traccar#3317]
i'm installed traccar 3.1 in my server, i using mozilla browser, but marker not showing in the map, can you help me?

![image](https://user-images.githubusercontent.com/4077667/27846002-4b6be6fa-615e-11e7-86b0-97d39fe5e81a.png)

thanks

**Comment by tananaev** [comment]
Looks like your devices are not connected to the server and never reported any location. Not sure why you expect to see something. You should check this guide:

https://www.traccar.org/troubleshooting/

**Comment by saepul8** [comment]
log result like this..., port number is correct

![image](https://user-images.githubusercontent.com/4077667/27847254-c9a948fc-6166-11e7-8d6b-4316e1490611.png)

**Comment by tananaev** [comment]
There is nothing in your log, which means that server doesn't receive any data from the device(s).

**Comment by saepul8** [comment]
i was check this  https://www.traccar.org/troubleshooting/ , still not work.

what should i do?

**Comment by tananaev** [comment]
If you followed troubleshooting guide, then you should already know what the problem is. If you followed it and still have no clue, please write every step that you did and the result that you see.

**Comment by saepul8** [comment]
i'm do this

1. install latest JVM (Java version "1.8.0_131")
2. install traccar v.3.11 and restrart
3. running traccar service (service running well)
4. register and then i'm login to web app 
5. add device serial number (IMEI)

i'm using OS MS Windows 2008, with VPS it success but  in my server not work.

thankyou

**Comment by tananaev** [comment]
No, describe steps from troubleshooting guide, not installation steps.

**Comment by saepul8** [comment]
i checked the log , not HEX receive, only this

2017-07-05 11:52:30  INFO: Operating system name: Windows Server 2008 version: 6.0 architecture: x86
2017-07-05 11:52:30  INFO: Java runtime name: Java HotSpot(TM) Client VM vendor: Oracle Corporation version: 25.131-b11
2017-07-05 11:52:30  INFO: Memory limit heap: 247mb non-heap: 0mb
2017-07-05 11:52:30  INFO: Character encoding: UTF-8 charset: UTF-8
2017-07-05 11:52:30  INFO: Version: 3.11-SNAPSHOT
2017-07-05 11:52:34  INFO: Starting server...


i check device using other listener, data is received

**Comment by tananaev** [comment]
I won't be able to help you if you can't even follow simple steps.

**Comment by saepul8** [comment]
i see, i wass follow that simple steps, but still not work for my server.  

thankyou

**Comment by tananaev** [comment]
I'm going to repeat myself last time. Write down every step that you did, including what you did and what was the result. In as much detail as possible.

**Comment by saepul8** [comment]
i'm using port number 5023 for my unit GT06, not receive data. and then i change the port with 5011
i receive this : 
 
2017-07-05 12:49:08 DEBUG: [82E084DC: 5011 < 127.0.0.1] HEX: 28424547494e0f000099001107050c3108044e4f4f50454e4429
2017-07-05 12:49:14 DEBUG: [82E084DC: 5011 < 127.0.0.1] HEX: 28424547494e0f00009a001107050c310e044e4f4f50454e4429
2017-07-05 12:49:20 DEBUG: [82E084DC: 5011 < 127.0.0.1] HEX: 28424547494e0f00009b001107050c3114044e4f4f50454e4429
2017-07-05 12:49:26 DEBUG: [82E084DC: 5011 < 127.0.0.1] HEX: 28424547494e0f00009c001107050c311a044e4f4f50454e4429
2017-07-05 12:49:32 DEBUG: [82E084DC: 5011 < 127.0.0.1] HEX: 28424547494e0f00009d001107050c3120044e4f4f50454e4429
2017-07-05 12:49:38 DEBUG: [82E084DC: 5011 < 127.0.0.1] HEX: 28424547494e0f00009e001107050c3126044e4f4f50454e4429

**Comment by tananaev** [comment]
Those messages are NOT from your device, so...
> it means that server doesn’t receive any data from your device. Possible reasons:
>
> Server is not visible from the Internet. Use port check tool to verify.
>
> Device is configured incorrectly or there is some problem with the device.
